lms/djangoapps/instructor/views/api.py

```diff
@@ -3601,39 +3601,6 @@ def _get_branded_email_template(course_overview):
     return template_name
 
 
-def _convert_schedule_to_utc_from_local(schedule, browser_timezone, user):
```

Comment:
Moved to the `instructor_task` app, longer explanation in the `instructor_task`'s `api.py` file.